Lib/gftools/builder/recipeproviders/googlefonts.py

```diff
@@ -193,8 +194,6 @@ def fontmake_args(self, source, variable=False):
             args += " --keep-direction"
         if self.config.get("removeOutlineOverlaps") is False:
             args += " --keep-overlaps"
-        if self.config.get("expandFeaturesToInstances"):
-            args += " --expand-features-to-instances"
```

Comment:
This is the approach in the release as of when I made the PR, right? If so, then yes, I’m 100% sure it wasn’t working. I can test it again, though, if it seems likely that it may have changed since then.

Reply:
That line feels right - I think I'd like to understand why it's not working before trying something else, because understanding why not might lead us to the right answer.